packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -24,20 +24,33 @@ export const _videoGalleryRemoteParticipantsMemo = (
     return [];
   }
   return memoizedAllConvertRemoteParticipant((memoizedFn) => {
-    return Object.values(remoteParticipants)
-      .filter((participant: RemoteParticipantState) => {
-        return participant.state !== 'InLobby';
-      })
-      .map((participant: RemoteParticipantState) => {
-        return memoizedFn(
-          toFlatCommunicationIdentifier(participant.identifier),
-          participant.isMuted,
-          checkIsSpeaking(participant),
-          participant.videoStreams,
-          participant.state,
-          participant.displayName
-        );
-      });
+    return (
+      Object.values(remoteParticipants)
+        /**
+         * hiding participants who are inLobby, idle, or connecting in ACS clients till we can admit users through ACS clients.
+         * phone users will be in the connecting state until they are connected to the call.
+         */
+        .filter((participant: RemoteParticipantState) => {
+          return (
+            (participant.state !== 'InLobby' && participant.state !== 'Idle' && participant.state !== 'Connecting') ||
+            participant.identifier.kind === 'phoneNumber'
+          );
+        })
+        .map((participant: RemoteParticipantState) => {
+          const state =
```

Comment:
given this odd logic is in two places, might be worth a helper function for the whole filter & map